src/Avalonia.Base/Threading/DispatcherTimer.cs

```diff
@@ -117,6 +117,11 @@ public object Tag
             set;
         }
 
+        /// <summary>
+        /// Stops the timer and disposes of all unmanaged resources.
+        /// </summary>
+        public void Dispose() => _timer?.Dispose();
```

Comment:
Add a Dispose(bool) method and have Dispose() and the finalizer call it (passing in true and false, respectively)

Reply:
Also add GC.SupressFinalize(this) in Dispose()